package/utils/nand-utils/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nand-utils
+PKG_REV:=d9076259b7f2654599bdf3416ec8ce0ad3b22610
+PKG_VERSION:=1.5.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=git://git.infradead.org/mtd-utils.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=$(PKG_REV)
+
+PKG_INSTALL:=1
+
+PKG_BUILD_DEPENDS:=util-linux liblzo zlib
+
+PKG_LICENSE:=GPLv2
+PKG_LICENSE_FILES:=
+
+PKG_MAINTAINER:=John Crispin <blogic@openwrt.org>
```

Comment:
Oh sorry. I use ubi-utils Makefile as example and forgot to fix this line.
kernel2minor utility pack ELF linux kernel to yaffs2 image with OOB data(yaffs2 tags + ecc) and for write such images we need nandwrite as the MTD utility is not able to write images with OOB.
And since we are working with nand flash the nandtest utility will not be superfluous(for bad blocks check).